packages/forms-web-app/src/views/components/section/results.njk

```diff
@@ -1,3 +1,12 @@
+{% macro getFileName(description, personalName, path) %}
```

Comment:
Following on from the above comment; if the logic was moved to the controller then this logic could live there and be tested.

Reply:
the meaty bit of the logic has a unit test, but I see what you mean, it would be nice to have a test for the whole thing